doc/user/zebra.rst

```diff
@@ -358,6 +370,139 @@ default) should the specified gateways not be reachable. E.g.:
 
 .. _multicast-rib-commands:
 
+VRF (Virtual Routing and Forwarding)
```

Comment:
`Virtual Routing and Forwarding` not a parenthesized expansion of an acronym in a title